scss/main.scss

```diff
@@ -465,6 +465,13 @@ h1,h2,h3 {
   margin-bottom: 2rem;
 }
 
+.workwithus {
+  padding-top: 3rem;
```

Comment:
I don't believe this needs extra padding:

**PR:**
<img width="1051" alt="Screen Shot 2021-02-11 at 8 58 13 AM" src="https://user-images.githubusercontent.com/9326234/107653713-8c76b600-6c47-11eb-8862-43f12b604da6.png">


**Design:**
<img width="427" alt="Screen Shot 2021-02-11 at 8 58 39 AM" src="https://user-images.githubusercontent.com/9326234/107653647-7963e600-6c47-11eb-9ffd-566cda6648e7.png">

Reply:
Got it fixed. Thanks for catching that!